Vectors.Rmd

```diff
@@ -502,7 +502,7 @@ structure(now_ct, tzone = "Europe/Paris")
 \index{durations|see {difftime}}
 \indexc{difftime}
 
-Durations, the amount of time between two dates or date times, are stored in difftimes. Difftimes are built on top of doubles, and have a units attribute that determines how the integer should be interpreted:
+Durations, the amounts of time between two dates or date times, are stored in difftimes. Difftimes are built on top of doubles, and have a units attribute that determines how the integer should be interpreted:
```

Comment:
Maybe:

Durations, which represent the amount of time between pairs of dates or date-times, are ...